pkg/k8shandler/desirednodestate.go

```diff
@@ -19,7 +19,7 @@ import (
 const (
 	elasticsearchCertsPath    = "/etc/openshift/elasticsearch/secret"
 	elasticsearchConfigPath   = "/usr/share/java/elasticsearch/config"
-	elasticsearchDefaultImage = "quay.io/openshift/logging-elasticsearch5"
+	elasticsearchDefaultImage = "docker.io/openshift/origin-logging-elasticsearch5"
```

Comment:
@jwforres what should we do here?  should we prefer 'quay.io' over docker?

Reply:
do we need a domain prefix at all?  Can we just use
```
 	elasticsearchDefaultImage = "openshift/origin-logging-elasticsearch5"
```
?